webpack/assets/javascripts/__tests__/foreman_tools-test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+describe('iconText', () => {
+  const tools = require('../foreman_tools');
+
+  it('creates a label with the right icon class', () => {
+    expect(tools.iconText('plus', '', 'patternfly'))
+    .toBe('<span class="patternfly patternfly-plus"/>');
+  });
+
+  it('adds a bold text next to the label', () => {
+    expect(tools.iconText('plus', 'foo', 'patternfly'))
+    .toBe('<span class="patternfly patternfly-plus"/><strong>foo</strong>');
+  });
+});
+
+describe('activateDatatables', () => {
+  it('calls this.DataTable when it finds a data-table=server', () => {
+    const $ = require('jquery');
```

Comment:
it calls '$.fn.DataTable', so the description in the describe function is confusing.